src/chapters/data-and-variables/values-and-data-types.rst

```diff
@@ -1,80 +1,91 @@
    console.log(typeof 17);
    console.log(typeof 3.14);
 
-Not surprisingly, strings are of type ``'string'`` while integers and floats are of type ``'number'``. 
+**Output**
 
-.. note::
+::
 
-	When we show the value of a string using the ``console.log()`` function, such as with ``console.log("Hello, World!")``, the quotes are no longer present. The value of the string is the sequence of characters inside the quotes. The quotes are only necessary to help JavaScript know what the value is, and are not part of the value itself.
+   string
+   number
+   number
 
-.. index:: operator
+Not surprisingly, strings are of type ``string`` while integers and floats are of type ``number``. Note that some JavaScript environments may print types names and strings with single quotes around them, as in ``'string'``, ``'number'``, and ``'hello'``.
 
-.. note::
+.. index:: expression, returns
 
-   ``typeof`` is new type of JS entity, an **operator**. It is similar to a function in that it carries out some kind of action, though the syntax is different from that of functions (notice there are no parentheses).
-   
-   We will learning about other operators in the comming sections.
+.. note:: Notice that ``console.log(typeof "Hello, World!");`` prints not ``typeof "Hello, World!"`` but something else entirely. In fact, ``typeof "Hello, World!"`` is an example of an **expression**, which we will learn about shortly. Briefly, expressions are code segments that can be reduced to a value. 
 
-In a JavaScript shell, it is not necessary to use the ``console.log()`` function to see the values shown above. The shell evaluates the ``typeof ...`` portion and automatically prints the result. For example, consider the shell session shown below. When we ask the shell to evaluate ``typeof "Hello, World!"``, it responds with the appropriate answer and then goes on to display the prompt for the next use.
+   We say that an expression **returns** a value. That is, ``typeof "Hello, World!"`` returns ``string``.
 
-::
+.. index:: operator
+
+.. note::
 
-   $ node
-   > typeof "Hello, World!";
-   'string'
-   > typeof 17;
-   'number'
-   > "Hello, World!";
-   'Hello, World!'
-   >
+   ``typeof`` is a JavaScript entity known as an **operator**. It is similar to a function in that it carries out some kind of action, though the syntax is different from that of functions (notice using ``typeof`` does not require parenthses).
+   
+   We will more about operators in upcoming sections.
 
-Note that in the last example, we simply ask the shell to evaluate the string "Hello, World".  The result is as you might expect, the string itself.
+There are data types other than string and number, including object and function, which we will learn about in future chapters.
 
 More On Strings
 ---------------
 
 What about values like ``"17"`` and ``"3.2"``? They look like numbers, but they are in quotation marks like strings.
 
-.. code-block:: js
+Run the following code to find out.
```

Comment:
Not sure about the output immediately following if you recommend running the code to find out.

Reply:
I converted this example to the "Try It!" format with a linked repl.it editor